system/jlib/jptree.cpp

```diff
@@ -6028,74 +6028,33 @@ static const char * skipWhitespace(const char * text)
     return text;
 }
 
-static const char * skipAsterisk(const char * text)
-{
-    if (*text=='*')
-        return skipWhitespace(text+1);
-    return text;
-}
-
-static const char * skipToNewline(const char * text)
-{
-    while (*text && (*text != '\r') && (*text != '\n'))
-        text++;
-    return text;
-}
-
-static const char * skipNewline(const char * text)
-{
-    if (*text == '\r')
-        text++;
-    if (*text == '\n')
-        text++;
-    return text;
-}
+// Unused Functions : -W-error-unused
+// static const char * skipAsterisk(const char * text)
+// {
+//     if (*text=='*')
+//         return skipWhitespace(text+1);
+//     return text;
+// }
+
+// static const char * skipToNewline(const char * text)
+// {
+//     while (*text && (*text != '\r') && (*text != '\n'))
+//         text++;
+//     return text;
+// }
+
+// static const char * skipNewline(const char * text)
+// {
+//     if (*text == '\r')
+//         text++;
+//     if (*text == '\n')
+//         text++;
+//     return text;
+// }
 
 void extractJavadoc(IPropertyTree * result, const char * text)
 {
-    //Skip a leading blank line
```

Comment:
Processing of documentation is being done in python ecldoc code (since I have added more markup to it, which will make it difficult to process in c++ and in this way, we will have single area where docstring processing occurs). I will replace the call with code directly in hqldesc. Should I remove this function then from here ?